projects/kernel-config/README.md

```diff
@@ -0,0 +1,38 @@
+## kernel config project
+
+The intent of the kernel config project is to demonstrate a better way to
+handle kernel config. Specifically:
+
+* support for arch and version specific config
+* make diffs as readable as possible
+* ensure that all of our config settings are kept after oldconfig
+
+We achieve the goals by:
+
+* having version-specific config in separate files, which are automatically
+  merged
+* only keeping track of visible symbols, only keeping track of a delta from
```

Comment:
what does "symbol" mean in this context. I may have a very narrow concept of symbol in my head...

further down you specifically call it "kconfig symbol" so might be worth using that on first use of symbol